### ISSUE TITLE
Module imports fail during run - failed install

### ISSUE THREAD

**Issue by callison96** [couler-proj/couler#142]
## Summary 

What happened/what you expected to happen?

Expected
- Run example code and no issues on install.

Actual
- Ran example code after install and got module import errors.

I can see in the setup.py file that this directory couler/proto is meant to be compiled but appears to be failing.

## Diagnostics

What is the version of Couler you are using? 

0.1.1rc7

What is the version of the workflow engine you are using?

argo-workflows = 3.5.1 

Any logs or other information that could help debugging?

```

ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-eadc1eb71d16> in <module>
----> 1 import couler.argo as couler
      2 from couler.argo_submitter import ArgoSubmitter
      3 
      4 
      5 def job_a(message):

c:\users\me\source\jupyter_notebook\env\lib\site-packages\couler\__init__.py in <module>
     12 # limitations under the License.
     13 
---> 14 from couler.argo import *  # noqa: F401, F403

c:\users\me\source\jupyter_notebook\env\lib\site-packages\couler\argo.py in <module>
     21 
     22 from couler.argo_submitter import ArgoSubmitter
---> 23 from couler.core import states  # noqa: F401
     24 from couler.core.config import config_workflow  # noqa: F401
     25 from couler.core.constants import *  # noqa: F401, F403

c:\users\me\source\jupyter_notebook\env\lib\site-packages\couler\core\states.py in <module>
     15 
     16 from couler.core import utils
---> 17 from couler.core.proto_repr import cleanup_proto_workflow
     18 from couler.core.templates import Workflow
     19 

c:\users\me\source\jupyter_notebook\env\lib\site-packages\couler\core\proto_repr.py in <module>
     14 from couler.core import states, utils  # noqa: F401
     15 from couler.core.templates.output import OutputArtifact, OutputJob
---> 16 from couler.proto import couler_pb2
     17 
     18 DEFAULT_WORKFLOW = None

ModuleNotFoundError: No module named 'couler.proto'

```

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritize the issues with the most 👍.

**Comment by terrytangyuan** [comment]
You'll need to run `python setup.py proto` before running `python setup.py install`. Could you try if this fixes the problem you are encountering?

**Comment by terrytangyuan** [comment]
@callison96 Could you try again? The problem should be fixed by #146.

**Comment by callison96** [comment]
@terrytangyuan Thanks working now, looking forward to testing it out.
